app/js/controller/member.js

```diff
@@ -73,17 +74,30 @@ angular.module('teamform-member-app', ['firebase'])
 				
 			}else {
 				window.history.back();		// future: the case user type url directly
-				console.log("no this event")
+				console.log("no this event");
 				return;				
 			}
 				
 			// update $scope
 			$scope.$apply();
-		});		
+		});	
 		
+		//get table information from database by tid		
+		var eid = getURLParameter("q");
+		var refPath = "tables";
```

Comment:
1) I think you should another variable name because i have used already.
2) for eid, may be you remove my eid , and put var eid = getURLParameter("q"); at the top. 
Thx